Objects/listobject.c

```diff
@@ -452,7 +452,7 @@ list_contains(PyListObject *a, PyObject *el)
     for (i = 0, cmp = 0 ; cmp == 0 && i < Py_SIZE(a); ++i) {
         item = PyList_GET_ITEM(a, i);
         Py_INCREF(item);
-        cmp = PyObject_RichCompareBool(PyList_GET_ITEM(a, i), el, Py_EQ);
+        cmp = PyObject_RichCompareBool(item, el, Py_EQ);
```

Comment:
@pablogsal 
Sorry, I found nit things when I work on creating a backported patch.
I applied this nit fix on 3.8 backport. https://github.com/python/cpython/pull/18204

Please re-review for it :)
Thank you!

Reply:
Great, thanks!